packages/repl/src/routes/+page.svelte

```diff
@@ -27,7 +28,9 @@
 </script>
 
 <main>
-	<Repl bind:this={repl} showAst />
+	{#if browser}
```

Comment:
Why did this have to change?

Reply:
Reverted. it does not have to change, I was just wondering why Firefox did not work with the REPL and changed some stuff and committed it by mistake